src/Symfony/Component/Config/Definition/ArrayNode.php

```diff
@@ -198,7 +198,7 @@ public function addChild(NodeInterface $node): void
     protected function finalizeValue(mixed $value): mixed
     {
         if (false === $value) {
-            throw new UnsetKeyException(\sprintf('Unsetting key for path "%s", value: %s.', $this->getPath(), json_encode($value)));
+            throw new UnsetKeyException(\sprintf('Unsetting key for path "%s", value: "false".', $this->getPath()));
```

Comment:
why the double qutes around? they weren't here before, isn't it? (same below)
```suggestion
            throw new UnsetKeyException(\sprintf('Unsetting key for path "%s", value: false.', $this->getPath()));
```

Reply:
Ah right, it's fixed now